doc/users/python_tutorial.rst

```diff
@@ -0,0 +1,83 @@
+Python Client Tutorial
+======================
+
+.. note:: See :ref:`Python Client API <users/python_api:Python Client API>` for complete full API specification.
+
+First, install the CCF Python package:
+
+.. code-block:: bash
+
+    $ pip install ccf
+
+Then, in the Python interpreter or new file:
+
+.. literalinclude:: ../../python/tutorial.py
+    :language: py
+    :start-after: SNIPPET: import_clients
+    :lines: 1
+
+Set the following CCF node variables:
+
+.. code-block:: python
+
+    host = "<node-rpc-host>"        # Node RPC address or domain
+    port = <node-rpc-port>          # Node RPC port
+    ca = "<path/to/network/cert>"   # Network certificate path
+
+.. note:: :ref:`When deploying a test network <users/deploy_app:Deploying an Application>`, use any node's IP address and port number. All certificates and keys can be found in the associated ``common_dir`` folder.
```

Comment:
I think we should specify that host is a string and port is a number. And we should drop the `RPC` terminology if we can. As far as a _client_ is concerned, this is just the node's `IP address or domain`. Unrelated - since we're trying so hard to look like a HTTPS server, we can probably default to port 443 (/no port) in the client?

Reply:
Using 443 for the RPC port makes perfect sense. Let me do that in a separate PR as I think `cchost` will need updating as well.